src/NServiceBus.AcceptanceTests/Serialization/When_configuring_custom_xml_namespace.cs

```diff
@@ -66,7 +66,7 @@ public IncomingMutator(Context scenarioContext)
 
                 public Task MutateIncoming(MutateIncomingTransportMessageContext context)
                 {
-                    var document = XDocument.Parse(Encoding.UTF8.GetString(context.Body));
+                    var document = XDocument.Parse(Encoding.UTF8.GetString(context.Body.ToArray()));
```

Comment:
I think GetString accepts ReadOnlySpan<byte>.

Reply:
I don't think it's worth tweaking the tests